frontend/its-our-call/src/components/MainMenu.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+interface  MainMenuProps {
+  hidden: boolean;
+}
+
+interface MainMenuState { }
+
+class MainMenu extends React.Component<MainMenuProps, MainMenuState> {
+  render() {
+    return (
```

Comment:
I would add an `if this.props.hidden return null` or else the `hidden` prop means nothing from within the component itself.